src/tests/sqlsmith/src/lib.rs

```diff
@@ -245,8 +246,12 @@ impl<'a, R: Rng> SqlGenerator<'a, R> {
         }
     }
 
-    fn gen_having(&self) -> Option<Expr> {
-        None
+    fn gen_having(&mut self) -> Option<Expr> {
```

Comment:
What if there's no group by? A having without group by?

Reply:
Ok, I did not consider about that. Then I pass an argument stating whether there is group by to this function. Thank for noticing it.